ConsistencyManager/DataStructures/ModelChange.swift

```diff
@@ -0,0 +1,44 @@
+// © 2016 LinkedIn Corp. All rights reserved.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import Foundation
+
+/**
+ This class is used to show which child models have changed. This object is usually in a dictionary: `[String: ModelChange]`.
+ The strings represent IDs and it shows what's changed in this ID. Either the ID has been deleted or updated.
+ If it's been updated, there are potentially several models that have updated if you are using projections.
+ */
+public enum ModelChange: Equatable {
+    /**
+     This indicates that this ID has been updated with a new model.
```

Comment:
It is kind of weird to reference `this ID` in the documentation here, because this type knows nothing of the ID. Maybe it should?

Assuming the current implementation, I would recommend describing/documenting this type without any assumption about how it would be used.

If you want to document the `[String: ModelChange]` data structure used elsewhere, create a typealias (or a proper type) somewhere and document that.